### ISSUE TITLE
Question / Proposal: Possible to send turn-by-turn instructions via BLE? [Android]

### ISSUE THREAD

**Issue by euphi** [abrensch/brouter#675]
Would it be possible to send turn-by-turn instructions on Bluetooth (LE) ? 

Usecase: I developed a bike computer (https://github.com/euphi/TRGB-BikeComputer) that can show komoot Route Instructions.

Till now, komoot is the only App I know that sends turn-by-turn info via Bluetooth.

I already opened a feature request for Osmand last year (https://github.com/osmandapp/OsmAnd/issues/16892), but I think it may be easier to have a standalone app that directly access brouter API.

Is it possible to connect to brouter API in parallel to Osmand (or other App?) and get same routing instructions?
If so, where can I find the API documentation to do this?
Would it be possible that brouter directly transmits routing info via BLE?

**Comment by abrensch** [comment]
Hi Euphi,

I think the place for such a functionality would be the navigation-app, the brouter-app is just a route-provider and it does not even access the GPS.If you do not get response from OsmAnd, then LocusMaps could be the better option? At least they are famliar with Bluetooth connectivity.

regards, Arndt

**Comment by euphi** [comment]
Yes, I had a wrong understanding how turn-by-turn navigation works. I thought there are frequent request from Osmand (or other Apps) to receive current instructions. But now I understand that there is one request for routing and brouter returns a GPX file that can be enriched with turn-by-turn instructions.

I'll have a look at the Aidl interface of osmand, it seems that at least the same functionality as in komoot should be possible (which is not sufficient for complicated instructions).
